apollo-router-core/src/query_planner/router_bridge_query_planner.rs

```diff
@@ -23,7 +23,7 @@ impl RouterBridgeQueryPlanner {
 
 #[async_trait]
 impl QueryPlanner for RouterBridgeQueryPlanner {
-    #[tracing::instrument(name = "plan", level = "debug", skip_all)]
+    #[tracing::instrument(skip_all, name = "plan", level = "debug")]
```

Comment:
:yarn: File unnecessarily touch in this PR

If possible I would like to keep the commits on main focus on their actual feature. If you want to re-order all the `skip_all` everywhere, you can make it on a separate PR.